pkgs/desktops/gnome-3/extensions/pop-os-shell/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, nodePackages, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "pop-os-shell";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "shell";
+    rev =  "${version}";
```

no need for interpolation, it's already a string.
```suggestion
    rev = version;
```